submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,199 @@
+              src="assets/icons/mail.png"
+              alt="mail icon"
+            />Mail</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/news.png"
+              alt="news icon"
+            />News</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/maps.png"
+              alt="map icon"
+            />Map</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/files.png"
+              alt="folder icon"
+            />Files</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/contacts.png"
+              alt="contacts icon"
+            />Contacts</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/tanslation.png"
+              alt="translate icon"
+            />Translate</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/videos.png"
+              alt="videos icon"
+            />Videos</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/search.png"
+              alt="search icon"
+            />Search</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/photos.png"
+              alt="two people: yellow and blue icon"
+            />Photo</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/apps.png"
+              alt="circle with four colors: blue, green, yellow, red icon"
+            />Apps</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/documents.png"
+              alt="documents icon"
+            />Document</a
+          >
+        </li>
+      </ul>
+      <input type="checkbox" id="load-more-btn" />
+      <label class="load-more" for="load-more-btn">
+        <p class="btn-text">More</p>
+      </label>
+      <ul class="hidden-icons">
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/chat.png"
+              alt="chat icon"
```

Comment:
The icon is a decorative image and for decorative images, the `alt` attribute value must be empty.